### ISSUE TITLE
translate

### ISSUE THREAD

**Issue by genbel2** [1833183060/autolisp#2]
kindly, I am not at all practical visual code, I am trying to learn; someone can explain to me how I translate Chinese and have the extension in the right language for me.  I saw that there is a file .txt I can translate, but how do I insert it into the extension
thank you

**Comment by Iron-Brutus** [comment]
Pushing,

totaly agree.

Trying to get in contact with author and would offer kind of help, but got no reply. Englisch translation of attached function reference would be very helpful. 

regards and thanks in advance

**Comment by genbel2** [comment]
I also replaced the translated data.csv file, but it still gives the result in Chinese

**Comment by Iron-Brutus** [comment]
Yeah, same with me, tried to manipulate the .csv file too without success.

seems that the source was kind of helpfile before AutoCad 2013, can't find it in the web.

**Comment by genbel2** [comment]
I can't understand you about it 
seems that the source was kind of helpfile before AutoCad 2013
it could also be that the chinese i think works at 2 bits per character
and then the.csv file is built to read like this !!! ???

in a few days i will try to communicate with a friend in china, i will update you
